uberfire-security/uberfire-security-management/uberfire-security-management-keycloak/README.md

```diff
@@ -17,114 +31,65 @@ Installation notes
 Step 1 - Dependencies
 ---------------------
 
-This provider uses the *KeyCloak Admin Client* (which uses KeyCloak's REST API services) to manage the different entities, so the following keycloak artifacts are required:                         
+This provider uses a custom RestEasy client that consumes the Keycloak remote admin services, so the following JAXRS, RestEasy 
```

Comment:
Yeah those deps apply for both impls, so I have added the comment that it applies for both